source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionContainer.test.tsx

```diff
@@ -12,7 +12,8 @@ import { lookupCodesSlice } from '@/store/slices/lookupCodes';
 import { systemConstantsSlice } from '@/store/slices/systemConstants/systemConstantsSlice';
 import { act, render, RenderOptions, waitFor } from '@/utils/test-utils';
 
-import { CompensationRequisitionFormModel, PayeeOption } from './models';
+import { PayeeOption } from '../../../models/PayeeOption';
```

Comment:
please use absolute paths for these unless the relative path is very short (parent sibling or child).

Reply:
updated